integration/src/fio/fio.ts

```diff
@@ -1,31 +1,42 @@
 import { bip32ToAddressNList, HDWallet, FioWallet, supportsFio } from "@shapeshiftoss/hdwallet-core";
 
 import { HDWalletInfo } from "@shapeshiftoss/hdwallet-core/src/wallet";
-
 import * as tx01_unsigned from "./tx01.unsigned.json";
 import * as tx02_signed from "./tx02.signed.json";
 
 const MNEMONIC12_NOPIN_NOPASSPHRASE = "alcohol woman abuse must during monitor noble actual mixed trade anger aisle";
+const MNEMONIC12_NOPIN_NOPASSPHRASE2 = "all all all all all all all all all all all all";
 
 const TIMEOUT = 60 * 1000;
 
-export function fioTests(get: () => { wallet: HDWallet; info: HDWalletInfo }): void {
+export function fioTests(get: () => { wallet: HDWallet; info: HDWalletInfo; wallet2: HDWallet }): void {
```

Comment:
confused on wallet2 ❓

Reply:
ok kept going and answered my question. Simulating a payment request from wallet to another. 👍

Reply:
maybe call them payerWallet and payeeWallet for clarity?